src/main/java/frc/robot/subsystems/climber/commands/Climb.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.climber.Climber;
+
+/**
+ * The command lifts the robot up by a given number of meters.
+ */
+public class Climb extends CommandBase {
+
+    private final Climber climber;
+    private double height;
+
+    public Climb(Climber climber, double height) {
+        this.climber = climber;
+        this.height = height;
+        addRequirements(climber);
+    }
+
+    @Override
+    public void initialize() {
+        new AutoClimb(climber);

```

Comment:
Use a command group for the whole sequence instead. 
I also don't think it'd work because both `AutoClimb` and `Climb` require the climber.